### DISCUSSION TITLE
How to create point constraint with world space point and single body?

### DISCUSSION THREAD

**Discussion by tksuoran** [jrouwe/JoltPhysics#252] [Q&A] [answered]
I looked at the constraints provided by Jolt and I did not find a simple way to create a single body point constraints with world space point.

As w workaround, I tried to make a dummy rigid body to use with the constraint. But there also does not seem to be a dummy shape class that I could use for this.

Please advice what would be the correct approach to this. Thank you!

**Comment by jrouwe** [comment, marked as answer]
Use Body::sFixedToWorld for the first body in your constraint
